web/src/main/java/com/microsoft/applicationinsights/web/internal/httputils/ApplicationInsightsServletExtractor.java

```diff
@@ -29,7 +29,7 @@
     private static final String USER_AGENT_HEADER = "User-Agent";
 
     @Override
-    public String getUrl(HttpServletRequest request) {
+    public String getUniformResourceLocator(HttpServletRequest request) {
```

Comment:
Given that we want to follow https://javaee.github.io/javaee-spec/javadocs/javax/servlet/http/HttpServletRequest.html#getRequestURI, we want to make the name consistent here.

Reply:
sure thing.